### ISSUE TITLE
this is frustrating

### ISSUE THREAD

**Issue by DizzyDez** [DizzyDez/responsive-infographic#4]
Now that I've figured out the buttons on the bottom of my info graphic, I thought the top should be done too. Can you not use the id as a selector? because when I target `.boy`the hover works fine (making it all appear, but I'll work on that later) but not when it is `#star`. Can I not use the id'd stars as buttons? I was hoping to put a hover on them like the other ones that turned out so well.

I was also hoping to target the info the same way, through `:nth-child(2)`
@thomasjbradley

**Comment by thomasjbradley** [comment]
Yep, using `#star` should work; where in the code are you trying to target—I don't see it anywhere?
